docs/software/Containerizationon.md

```diff
@@ -100,7 +100,7 @@ Alternatively, users wishing to bind their entire CURC filesystem to a container
 apptainer run -B /projects/$USER,/pl/active,/scratch/alpine/$USER sample-image.sif
 ```
 
```

Comment:
The naming of this file has ALWAYS annoyed me. Can we change `docs/software/Containerizationon.md` so that containerization is spelled correctly i.e. `docs/software/Containerization.md`?  You will need to modify any associated documentation that refers to this name.